config/locales/en.yml

```diff
@@ -133,6 +133,9 @@ en:
       one_login_identity:
         name: One Login identity check
         title: Confirm the claimant made the claim
+      alternative_identity_verification:
+        name: "Alternative identity verification"
+        title: "Confirm the provider has verified the claimant's identity"
```

Comment:
smartquote

Reply:
Will fix this separately, got a few branches based off this one!